x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -67,10 +67,14 @@ export const AllCasesList = React.memo<AllCasesListProps>(
       owner: hasOwner ? owner : availableSolutions,
     };
 
-    const { queryParams, setQueryParams, filterOptions, setFilterOptions } = useAllCasesState(
-      isSelectorView,
-      initialFilterOptions
-    );
+    const {
+      queryParams,
+      setQueryParams,
+      filterOptions,
+      setFilterOptions,
+      clearFilterOptions,
+      defaultFiltersSelected,
```

Comment:
nit: While I was reading the code I thought that `defaultFiltersSelected` was an object. Maybe we can rename it to `areDefaultFiltersSelected` or similar to make it clearer that this is a boolean.